apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.story.jsx

```diff
@@ -59,44 +64,44 @@ const createStoreWithoutLessonPlan = () => {
   return store;
 };
 
-export default storybook => {
-  if (IN_UNIT_TEST) {
-    allowConsoleWarnings();
-  }
+if (IN_UNIT_TEST) {
+  allowConsoleWarnings();
+}
+
+//
+// TEMPLATE
+//
+
+const TemplateWithLessonPlan = args => (
+  <Provider store={createStoreWithLessonPlan()}>
+    <ActivitiesEditor {...args} />
+  </Provider>
+);
+
+const TemplateWithoutLessonPlan = args => (
+  <Provider store={createStoreWithoutLessonPlan()}>
+    <ActivitiesEditor {...args} />
+  </Provider>
+);
+
+//
+// STORIES
+//
+
+export const ForLessonWithLessonPlan = TemplateWithLessonPlan.bind({});
+ForLessonWithLessonPlan.args = {
+  hasLessonPlan: true,
+  allowMajorCurriculumChanges: true
+};
+
+export const ForLessonWithoutLessonPlan = TemplateWithoutLessonPlan.bind({});
+ForLessonWithoutLessonPlan.args = {
+  hasLessonPlan: false,
+  allowMajorCurriculumChanges: true
+};
 
-  storybook.storiesOf('ActivitiesEditor', module).addStoryTable([
-    {
-      name: 'ActivitiesEditor For Lesson With Lesson Plan',
-      story: () => (
-        <Provider store={createStoreWithLessonPlan()}>
-          <ActivitiesEditor
-            hasLessonPlan={true}
-            allowMajorCurriculumChanges={true}
-          />
-        </Provider>
-      )
-    },
-    {
-      name: 'ActivitiesEditor For Lesson Without Lesson Plan',
-      story: () => (
-        <Provider store={createStoreWithoutLessonPlan()}>
-          <ActivitiesEditor
-            hasLessonPlan={false}
-            allowMajorCurriculumChanges={true}
-          />
-        </Provider>
-      )
-    },
-    {
-      name: 'ActivitiesEditor when major changes are not allowed',
-      story: () => (
-        <Provider store={createStoreWithoutLessonPlan()}>
-          <ActivitiesEditor
-            hasLessonPlan={true}
-            allowMajorCurriculumChanges={false}
-          />
-        </Provider>
-      )
-    }
-  ]);
+export const WhenMajorChangesNotAllowed = TemplateWithoutLessonPlan.bind({});
+WhenMajorChangesNotAllowed.args = {
+  hasLessonPlan: true,
```

Comment:
Hi! Was this one meant to use the TemplateWithoutLessonPlan if hasLessonPlan = true? And would it be possible to pull that argument up into the template for both?

Reply:
hm good callout, my template names are confusing here. I'm not sure what this was intended to show, but the original `ActivitiesEditor when major changes are not allowed` test uses `createStoreWithoutLessonPlan` and `hasLessonPlan={true}`.

Reply:
do you think f191273 makes it any clearer?

Reply:
Here's what these look like:

![image](https://user-images.githubusercontent.com/1382374/210863342-ddf9e971-8996-4d20-91c7-24d15b816662.png)
![image](https://user-images.githubusercontent.com/1382374/210863391-a6dfd609-2666-4698-b031-d0d9fae539af.png)
![image](https://user-images.githubusercontent.com/1382374/210863431-035e9f8e-cb2d-4894-a96c-703fd28c6d4e.png)

Reply:
Thanks for the picture context!